doc/usecases/mount/UC_list_values.md

```diff
@@ -0,0 +1,40 @@
+# Use Case: List all configuration keys and values from all currently mounted data sources
```

Comment:
I don't know which functionality this is? Is this `export`? Why does it belong to the mounting use cases?